### ISSUE TITLE
Documentation is outdated on pythonhosted.org

### ISSUE THREAD

**Issue by septatrix** [pyserial/pyserial#536]
The documentation on [pythonhosted (3.0)](https://pythonhosted.org/pyserial/) is not up to date compared to the one on [readthedocs (3.4)](https://pyserial.readthedocs.io/en/latest/pyserial_api.html). The documentation on pythonhosted should be updated and IMO one of these sites should get shut down or redirect to the other one to prevent similar problems in the future.

**Comment by tobixen** [comment]
I came in here through google.  I have the same problem in my project, ref https://github.com/python-caldav/caldav/issues/95

I'll do more research on this one and update here if I find any solution to it.

**Comment by tobixen** [comment]
https://stackoverflow.com/questions/6521931/how-to-delete-documentation-from-pypi seems relevant.

**Comment by kurtmckee** [comment]
@septatrix pythonhosted.org cannot be updated. The answer that @tobixen linked to is 14 years old and there is no longer an interface to upload replacement documentation. 

@zsquareplusc @andrewleech Please close this issue. It's not possible to update the pyserial documentation on pythonhosted.org.

**Comment by septatrix** [comment]
Once can still reach out to PyPA via mail. I am quite positive that they would help to eliminate confusion

**Comment by dpgeorge** [comment]
IMO the docs at pythonhosted need to be either taken down, or redirected to the readthedocs page.  Because it's confusing if you land at pythonhosted.

**Comment by kurtmckee** [comment]
> the docs at pythonhosted need to be either taken down, or redirected to the readthedocs page

I agree (and wish I could do the same with multiple projects of my own that are also 15 and 20 years old and have old docs on pythonhosted.org). However, there is no way to do either of those things.

This bug is five years old and cannot be resolved so I recommend closing this issue.
